src/SetHandler.cpp

```diff
@@ -118,9 +104,6 @@ int SetHandler::getOptionalFields(const std::string& headerLine,
     } else if ((optionalHeader == "rt" || optionalHeader == "retentiontime")) {
       optionalFields.push_back(RETTIME);
       hasRt = true;
-    } else if (DataSet::getCalcDoc() && (optionalHeader == "dm" || optionalHeader == "deltamass")) {
-      optionalFields.push_back(DELTAMASS);
-      hasDm = true;
```

Comment:
A "dm" column will not break the run, however, it will introduce an unnececair feature. We keep the rt feature for other reasons (compatibility with fragpipe).